packages/template-retail-react-app/app/theme/components/base/input.js

```diff
@@ -21,11 +21,17 @@ export default {
         }
     },
     variants: {
+        outline: {
+            field: {
+                borderColor: 'gray.500'
```

Comment:
Gray 500 is the minimum value of blue colors that satisfies the constrast ratio 3:1 against #ffffff background
![image](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/723008a3-a159-42f2-9117-1ae4b68b48c0)